schema.ts

```diff
@@ -839,11 +839,17 @@ export type ObjectSchemaDefinitionType<
 > = ObjectSchemaType<T>;
 
 // This is basically the same as an ObjectSchemaDefinition but includes an Identity that may be injected
-// at upload time (for static schemas) or at runtime (for dynamic schemas). This is the type that is
-// used on the `coda` side for implementations, but should not need to be used by pack makers.
```

Comment:
Should we keep the comment line about this being used only in the `coda` implementation and that this isn't a type relevant to pack makers?

Reply:
Reverted! I got confused about the 2 different `identity` fields before.